documentation/docs/01-project-structure.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Project Structure
+---
+
+The basic structure of a SvelteKit project is:
+
+```sh
+my-app/
+ |
+ +---- .svelte-kit/                 # (generated at dev)
+ +---- build/                       # (generated at build)
+ +---- src/
+ |       |
+ |       +---- lib/                 # (optional)
+ |       +---- params/              # (optional)
+ |       +---- routes/
+ |       +---- app.css              # (optional)
+ |       +---- app.d.ts             # (optional)
+ |       +---- app.html
+ |       +---- hooks.js             # or 'hooks/' (optional)
+ |       +---- service-worker.js    # (optional)
+ +---- static/
+ +---- tests/                       # (optional)
+ +---- .eslintrtc.cjs               # (optional)
+ +---- .gitignore                   # (optional)
+ +---- .npmrc
+ +---- .prettierrc                  # (optional)
+ +---- jsconfig.json                # or 'tsconfig.json' (optional)
+ +---- package.json
+ +---- playwright.config.js         # (optional)
+ +---- svelte.config.js
+```
+
+If using TypeScript, the scripts files in the `src/` directory will have a `.ts` file extension.
+
+### package.json
+
+The `package.json` file is a standard file where you can configure your `dependencies` and `devDependencies`. [Read more in the npm docs](https://docs.npmjs.com/cli/v7/configuring-npm/package-json).
+
+SvelteKit is ESM-first so we set `"type": "module"`. In the `"scripts"` property, you can see scripts used for building the project, running the dev server, packaging a library (optional), and running other tools for linting, formatting, type checking, testing etc.
+
+### svelte.config.js
+
+Contains [configuration](configuration) for the SvelteKit project.
+
+### static/
+
+Holds your static assets. These are files that are not processed by Vite but are just copied to the build folder by the adapter when building the application.
+
+### src/
+
+Holds the Svelte, JS/TS, and other source files for your app.
+
+#### app.html
+
+This is the entry point to your application. It contains placeholders like `%svelte.head%`, `%svelte.body%`, and `%svelte.assets%`. These placeholders will be removed and replaced by actual content at build or SSR.
+
+`%svelte.head%` is replaced with `<link>` and `<script>` tags that power the application as well as content from the [`<svelte:head>`](https://svelte.dev/docs#template-syntax-svelte-head) tags defined in your layouts and pages.
+
+`%svelte.body%` is where the HTML for your application is rendered.
+
+`%svelte.assets%` is replaced by the [assets base path](configuration#paths).
+
+#### routes/
+
+Contains your [pages](routing#pages), [endpoints](routing#endpoints), and [layouts](layouts) and can also hold private module files. SvelteKit use file-based routing, so the file path of the pages and endpoints in this folder map to the URL which renders them.
```

Comment:
Reading through current documentation, I was wondering if we should also mention [Standalone endpoints](https://kit.svelte.dev/docs/routing#endpoints-standalone-endpoints) because of their related `data/` folder.
Thoughts?

```suggestion
Contains your [pages](routing#pages), [endpoints](routing#endpoints), [standalone endpoints](routing#endpoints-standalone-endpoints), [layouts](layouts) and can also hold private module files. SvelteKit uses file-based routing, so the file path of the pages and endpoints in this folder maps to the URL which renders them.
```

BTW, there are some errors in the text:
- a superfluous "and" in "…,~and~ layouts and can also hold private module files."
- a wrong declination for the _use_ verb in "…SvelteKit use***s*** file-based routing,…:
- a wrong declination for the _map_ verb in "…map***s*** to the URL…"

They are all fixed in the proposed suggestion.